src/migrations/locale.ts

```diff
@@ -0,0 +1,94 @@
+export const WEEK_START = {
```

Comment:
Could add a FIXME to remove this map once FF starts supporting https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getWeekInfo